package/yast2-samba-client.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Fri Jan 18 19:38:21 UTC 2019 - David Mulder <dmulder@suse.com>
+
+- Use a cldap ping to search for the join server; (bsc#1122534).
+- 4.0.5
```

Comment:
As it will diverge from SLE-15-GA branch, please bump minor version.
```suggestion
- 4.1.0
```

Reply:
Rest LGTM, I'll merge after the suggested version change